crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/meta_net_task.rs

```diff
@@ -499,6 +508,10 @@ impl MetaNetTask {
                             continue;
                         }
 
+                        // Now that the host is holding the op, remove it from the fetch pool. Any sooner and we might queue the op for fetching again.
+                        // TODO we haven't waited for the op to finish validation, need to be careful about limbo state here?
```

Comment:
```suggestion
                        // Now that the host is holding the op, remove it from the fetch pool. Any sooner and we might queue the op for fetching again.
                        // We don't need to wait for validation to complete, at least with respect to gossip, because we don't ask for unvalidated
                        // ops during gossip. (See https://github.com/holochain/holochain/blob/3447d7a1a55980b79c91fd00bb09a31219dc96e7/crates/holochain/src/conductor/kitsune_host_impl/query_region_set.rs#L15-L22)
```